app/graphql/resolvers/rdf.ts

```diff
@@ -33,7 +32,37 @@ import {
 } from "@/rdf/query-cube-metadata";
 import { unversionObservation } from "@/rdf/query-dimension-values";
 import { queryHierarchy } from "@/rdf/query-hierarchies";
-import { searchCubes } from "@/rdf/query-search";
+import { SearchResult, searchCubes } from "@/rdf/query-search";
+
+const sortResults = (
+  results: SearchResult[],
+  order: DataCubeResultOrder | undefined | null,
+  locale: string | undefined | null
+): SearchResult[] => {
+  const getCube = (r: SearchResult) => r.dataCube.data;
+  switch (order) {
+    case DataCubeResultOrder.TitleAsc:
+      results.sort((a, b) =>
+        getCube(a).title.localeCompare(getCube(b).title, locale ?? undefined)
+      );
+      break;
+    case DataCubeResultOrder.CreatedDesc:
+    case undefined:
+    case null:
+      results.sort((a, b) => {
+        const ra = getCube(a).datePublished || "0";
+        const rb = getCube(b).datePublished || "0";
+        return descending(ra, rb);
+      });
+      break;
+    case DataCubeResultOrder.Score:
+      break;
```

Comment:
Results are sorted be score by default, right?

Reply:
Yes, when they enter this function, they are already sorted.